config/zones/UA.yaml

```diff
@@ -90,7 +90,7 @@ emissionFactors:
       _comment: estimated yearly share of solar & wind in this mixed renewable category
         based on installed capacity in 2018
       _url: https://en.wikipedia.org/wiki/Renewable_energy_in_Ukraine
-      source: renewable mix; assumes 50% solar, 50% wind
+      source: assumes 50% solar, 50% wind renewable mix
```

Comment:
According to figures from the [latest currently-available iteration of the same Wikipedia source page](https://en.wikipedia.org/w/index.php?title=Renewable_energy_in_Ukraine&oldid=1153471433#History), in Y2021 the ratio of `solar:wind` capacity was `6227:1673`.

However: those figures predate the conflict in Ukraine that began in Y2022 and so the numbers may not correspond to the situation more recently.  Note also that data for Ukraine is currently unavailable on Electricity Maps (the parser has been down for a while; I'm not cross-linking the relevant issue thread here because it's not relevant to most of this pull request).

Reply:
Also: installed capacity ratio doesn't necessarily indicate actual output capacity.  So even though the capacity ratio may have changed, the ratio itself may not be suitable to map 1-1 to the percentages assumed here.